docs/services/education-consultant.md

```diff
@@ -0,0 +1,31 @@
+# Education Consultant
+
+The Education Consultant has experience with designing curriculum and creating positive learning spaces for data science education.
+
+## Goals
+
+* Helping all learners feel included in the educational and project spaces at hackweeks.
+* Educating trainers in practices for building interactive and engaging tutorial content. 
+
+## Specific Activities
+
+* Offer a series of tutorials to hackweek organizers for best practices in data science pedagogy.
+* Facilitate peer feedback sessions for hackweek tutorial leads as they develop content.
```

Comment:
Add:

* Provide content editing with a pedagogical lens for tutorial leads as they develop written content.